BUILD

```diff
@@ -28,3 +28,21 @@ gazelle(
 
 # This could be any file, used as an anchor point for the directory in tests
 exports_files(["README.md"])
+
+go_path(
+    name = "all_srcs",
+    deps = [
+        "//go/tools/builders:asm",
```

Comment:
How is this list generated? Should it include tests and libraries?

Might be good to add a script or command in the comments to generate this. Maybe `bazel query` will work after toolchains lands?